### PR TITLE
[virtualization] Document updates of VM spec

### PR DESCRIPTION
**Pull request by kvaps**
## Description

Document updates of VM spec

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: virtualization
type: chore
summary: Documentation updates of VM spec.
impact_level: low
```

### CODE REVIEW

File: modules/490-virtualization/docs/FAQ_RU.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 title: "Модуль virtualization: FAQ"
 ---
 
+## Как применить изменения к виртуальной машине?
+
+На данный момент изменения в шаблоне виртуальной машины не применяются к запушенным инстансам автоматически.  
+Для того чтобы применить изминения, выполните удаление запущенного инстанса ВМ:
```
Comment (z9r5) [inline modules/490-virtualization/docs/FAQ_RU.md:8]:
```suggestion
Для того чтобы применить изменения, выполните удаление запущенного инстанса ВМ:
```

File: modules/490-virtualization/docs/FAQ_RU.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+
+```bash
+kubectl delete virtualmachineinstance <vmName>
+```
+
+Вновь созданный инстанс ВМ, будет включать все последние изменения из спеки ресурса VirtualMachine.
```
Comment (z9r5) [inline modules/490-virtualization/docs/FAQ_RU.md:14]:
```suggestion
Вновь созданный инстанс ВМ будет включать все последние изменения из ресурса [VirtualMachine](cr.html#virtualmachine).
```

File: modules/490-virtualization/docs/FAQ.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+## How to apply changes to the virtual machine spec?
+
+Currently, changes to the VM specification are not applied to running instances automatically.
+To apply the changes, delete the running VM instance:
+
+```bash
+kubectl delete virtualmachineinstance <vmName>
+```
+
+The newly created VM instance will include all the latest changes from the VirtualMachine spec.
```
Comment (z9r5) [inline modules/490-virtualization/docs/FAQ.md:14]:
```suggestion
The newly created VM instance will include all the latest changes from the [VirtualMachine](cr.html#virtualmachine) custom resource.
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [4186](https://github.com/deckhouse/deckhouse/pull/4186) to the branch [release-1.44](https://github.com/deckhouse/deckhouse/tree/release-1.44) successful!

**Pull request merged**